packages/utils/src/open-search/semantic-ingest.ts

```diff
@@ -26,20 +36,30 @@ const metriportAPI = new MetriportMedicalApi(apiKey, {
   timeout: 120_000,
 });
 
+const outputFolderName = `semantic-ingest`;
+initRunsFolder(outputFolderName);
+const getFolderName = buildGetDirPathInside(outputFolderName);
+const outputFilePrefix = getFolderName(`${cxId}_${patientId}`);
+const outputFilePath = outputFilePrefix + ".ndjson";
+
 async function main() {
   await sleep(50); // Give some time to avoid mixing logs w/ Node's
   const startedAt = Date.now();
-
+  initSentry();
```

Comment:
do we want to commit this or is this left over from testing?

Reply:
intentional, so we can test sending stuff to sentry